exercises/isogram/HINT.md

```diff
@@ -0,0 +1,7 @@
+If you find that `testWorksWithGermanLetters` fails even though you think you've implemented it correctly,
+this could be because of the [JVM](https://en.wikipedia.org/wiki/Java_virtual_machine) encoding. Try setting this encoding in the environment variables on
+your machine:
+1. Go to Computer -> Properties -> Advanced System Settings
```

Comment:
If I understand correctly, this could also be an issue on non-Windows machines, where environment variables are set differently. Should we (a) give per-platform instructions here, (b) give no implementation instructions here, or (c) do something else that I haven't thought of?